com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1351,7 +1362,7 @@ private void ConfigureSimulator()
 #endif
 #endif
 
-#if UNITY_EDITOR || DEVELOPMENT_BUILD
+#if UNITY_EDITOR || DEVELOPMENT_BUILD || MULTIPLAYER_TOOLS_1_1_0
```

Comment:
This is based on an assumption that this will be defined when MULTIPLAYER_TOOLS_1_1_0 is released which would contain NetSim. @becksebenius-unity @fluong6

Reply:
This would need to be added to the assembly definition as a conditional compiler directive, it won't happen automatically

Reply:
Should this `||` only occur if `NetSim` is actually included in the build? It's possible to have a release build without NetSim included right?